src/allmydata/test/web/test_web.py

```diff
@@ -841,63 +845,95 @@ def _check(res):
         d.addCallback(_check)
         return d
 
+    @inlineCallbacks
     def test_welcome(self):
```

Comment:
I agree some of the tests are ... questionable at best. Definitely not opposed to deleting this one! (Mostly my strategy with "porting" has been to resist the urge to rip out the "refactoring chainsaw" since this branch is already huge).

Reply:
Okay, I'll delete it (and filed https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3280#ticket instead).